index.bs

```diff
@@ -3482,6 +3558,13 @@ The [=remote end steps=] with |session| and |command parameters| are:
             given |context nodes|, |selector|, |max depth|, |match type|,
             |ignore case|, |maximum returned node count| and |session|.
 
+      <dt>|type| is the string "<code>accessibility</code>"
+      <dd>
+         1. Let |selector| be |locator|["<code>value</code>"].
+
+         1. Let |result nodes| be a result of [=trying=] to [=locate nodes using accessibility attributes=]
```

Comment:
```suggestion
         1. Let |result nodes| be [=locate nodes using accessibility attributes=]
```

Since we're no longer returning a result type, we can't use `[=trying=]`.